app/views/layouts/readme.njk

```diff
@@ -1,3 +1,4 @@
+{% set nunjucksHtmlUsageMessage = '**If you’re using Nunjucks macros in production be aware that using  `html` arguments, or ones ending with `Html` can be a [security risk](https://en.wikipedia.org/wiki/Cross-site_scripting).**' %}
```

Comment:
Are there any components where this notice isn't being added? If not, is there any reason we couldn't just add this to this layout above the `componentArguments` block and save including this in every component?

Reply:
no, it's everywhere. updated it. thanks